suseconnect-ng.spec

```diff
@@ -32,6 +32,26 @@ BuildRequires:  go >= 1.16
 Conflicts:      SUSEConnect
 Obsoletes:      SUSEConnect
 Provides:       SUSEConnect
+%if 0%{?fedora} || 0%{?rhel} || 0%{?centos_version}
+Requires:       ca-certificates
+%else
+Requires:       ca-certificates-mozilla
+%endif
+Requires:       coreutils
+# ExclusiveArch from this package
+%ifarch %ix86 ia64 x86_64 %arm aarch64
+Requires:       dmidecode
+%endif
+# ExclusiveArch from this package
+%ifarch s390x
+Requires:       s390-tools
+%endif
+Requires:       systemd
```

Comment:
why is systemd required?

Reply:
hwinfo.go executes systemd-detect-virt which is in that rpm.